servers/src/main/java/tachyon/master/next/filesystem/meta/Inode.java

```diff
@@ -109,27 +130,18 @@ public synchronized long getParentId() {
     return mParentId;
   }
 
-  /**
-   * Get the pinned flag of the inode
-   *
-   * @return true if the inode is pinned, false otherwise
-   */
-  public synchronized boolean isPinned() {
-    return mPinned;
+  @Override
+  public synchronized int hashCode() {
+    return ((Long) mId).hashCode();
   }
 
   /**
-   * Get the last modification time of the inode
+   * Return whether the inode is deleted or not.
    *
-   * @return the last modification time, in milliseconds
+   * @return true if the inode is deleted, false otherwise
    */
-  public synchronized long getLastModificationTimeMs() {
-    return mLastModificationTimeMs;
-  }
-
-  @Override
-  public synchronized int hashCode() {
-    return ((Long) mId).hashCode();
+  public boolean isDeleted() {
```

Comment:
No callers for now. Put it here because (1) it might be used in the future (2) to get rid of the unused variable warning.